focus-areas/people/contributors.md

```diff
@@ -1,75 +1,98 @@
+   - Event participants
+   - IRC or forum authors
+- By project characteristics:
+   - By release cycle
+   - Programming languages of the project
+   - Role or function in the project (e.g., coding, organizing, mentoring)
+   - Timeframe of activity (e.g., identifying new contributors)
+
+### **Visualizations**
+1. **List of Contributors:**  
+   Display contributor names with engagement details.
+<img width="490" alt="IMG_5268" src="https://github.com/user-attachments/assets/456864b5-6632-4fda-93a6-0ada9ca10b45">
+ 
+  *Figure 1: List of contributors and engagement (Source, Year)*
+
+2. **Summary of Contributors:**  
+   A summary showing the total number of contributors.
+![IMG_5269](https://github.com/user-attachments/assets/1d60591c-3024-4068-82ed-241846c03ac7)
+
+   *Figure 2: Summary number of contributors (Source, Year)*
+
+4. **Growth Over Time:**  
+   Change in the number of active contributors over time.
+![IMG_5270](https://github.com/user-attachments/assets/c13de6dc-8f08-4d5d-b98b-dba0b3673dc8)
+
+    *Figure 3: Contributor growth over time (Source, Year)*
+
+6. **New Contributors:**  
+   A list sorted by the date of first contribution.
+   
+![IMG_5271](https://github.com/user-attachments/assets/22f02ff1-6023-4ec1-92a9-a7efdeb2e25f) 
+  
+   *Figure 4: List of new contributors (Source, Year)*
+
+### **Surveys and Interviews**
+Some contributor insights are less easily obtained from trace data and may require additional surveys or interviews. Sample questions include:
+- *Which contributors do not typically appear in lists of contributors?*
+- *Which contributors are often overlooked because their contributions are more “behind the scenes”?*
+- *Who are the community members you work with regularly?*
+
+Surveys with Likert scale or matrix formats can also provide insights into contributor activity levels.  
+Sample questions for community members:
+- *Likert scale [1-x]: I am contributing to the project.*
+- *Matrix survey item:* How often do you engage in the following activities in the project?
+   - Columns: Never, Rarely (less than once a month), Sometimes (more than once a month), Often (once a week or more)
+   - Rows: a) Contributing/reviewing code, b) Creating or maintaining documentation, c) Translating documentation, d) Participating in decision-making, e) Serving as a community organizer, f) Mentoring, g) Attending events, h) Participating through school or university computing programs, i) Participating through a program like Outreachy, Google Summer of Code, etc., j) Helping with the ASF operations (e.g., board meetings or fundraising)
+
+</details></span>
+
+## **References**
+None specified.
+
+## **Contributors**
+- Kevin Lumbard
+- Vinod Ahuja
+- Elizabeth Barron
+- Dawn Foster
+- Sean Goggins
+- Matt Germonprez
+- Yigakpoa L. Samuel (Ikpae)
+
+## **Additional Information**
+- To edit this metric please submit a Change Request here: https://github.com/chaoss/wg-common/blob/main/focus-areas/people/contributors.md
```

Comment:
@yigakpoa can you change the path to match the path of this file? (we changed the repo name a while back from wg-common to wg-metrics-development)

Reply:
Done, thank you